examples/selenium-container/src/test/java/SeleniumContainerTest.java

```diff
@@ -36,7 +36,7 @@
     @Rule
     public BrowserWebDriverContainer chrome = new BrowserWebDriverContainer()
                                                     .withCapabilities(new ChromeOptions())
-                                                    .withRecordingMode(RECORD_ALL, new File("target"));
+                                                    .withRecordingMode(RECORD_ALL, new File("build"));
```

Comment:
Does this potentially break the Maven example?

Reply:
It does, but the idea is to remove the Maven examples, since we really don't want to maintain both build systems for the examples.

Reply:
I see. Thanks for clarifying.